test/test_module_availability/test_main.cpp

```
@@ -0,0 +1,46 @@
+#include "TestUtil.h"
+#include "meshtastic/config.pb.h"
+#include "modules/ModuleAvailability.h"
+#include <Arduino.h>
+#include <unity.h>
+
+static void test_audio_available_only_on_esp32_sx1280_lora24()
+{
+    TEST_ASSERT_TRUE(isAudioModuleAvailable(true, true, false, meshtastic_Config_LoRaConfig_RegionCode_LORA_24));
+}
```

Cppcheck (2.21.0)

[error] `#error` Regenerate this file with the current version of nanopb generator.

(preprocessorErrorDirective)